src/prompt-storage.ts

```diff
@@ -26,7 +26,7 @@ export const rememberAuthToken = (
   token: string,
   expiresAt: Date,
 ) => {
-  Cookies.set(`bucket-token-${userId}`, `${channel}:${token}`, {
+  Cookies.set(`bucket-token-${userId}`, JSON.stringify([channel, token]), {
```

Comment:
How about storing an actual object with keys to avoid any mistakes in ordering or similar?

Reply:
Seconded

Reply:
done